content/blog/posts/c-stl-algorithms-series-part-1.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "C++ STL Algorithms Series - Part 1"
+date: 2022-07-13
+slug: c-stl-algorithms-series-part-1
+author: Tom Hulton-Harrop
+blog_img: ""
+full_img: ""
+---
+# Item 1: C++ STL Algorithms Series - Part 1
+
+## Topic
+
+An introduction to why you might want to start using the `<algorithm>` and `<numeric>` header more in your code.
+
+## Motivation
```

Comment:
```suggestion
## Motivation
```
Motivation and Deliberation headings seem like they were a part of the writing process rather than a part of the presentation.  Consider removing/replacing them for this series.

Reply:
I think the headers make these posts succinct and are reminiscent of some research abstracts/proposals I've seen in the past. Might seem formal but it's a stylistic choice and feels appropriate for the content.